spec/support/rb_bug.rb

```diff
@@ -1,5 +1,6 @@
 require 'ffi'
 
+# @api private
```

Comment:
@dkubb I meant `@api private`, I assume this is the best way to set a inherited default for `@api` YARD tags on the full mutant namespace. Setting this allows me to remove all the redundant `@api private` tags for individual methods / classes.

Mutant intentionally has no specified public API. The only stable thing is the CLI.